### DISCUSSION TITLE
Allow selecting any issues in "Linked issues"

### DISCUSSION THREAD

**Discussion by per1234** [community/community#12224] [Pull Requests]
The "**Link an issue from this repository**" dropdown menu of the "**Linked issues**" offers a small selection of issues from the repository; apparently what some algorithm has deemed relevant. I very frequently find that the issues I need to link are not in that list.

This approach makes what could be an extremely useful feature for repository maintainers almost worthless.

Just let me specify the issue number or URL.

---

UPDATE:

In order to avoid confusion, I should add that it looks like GitHub has renamed the section on the PR and issues page from "**Linked issues**" to "**Development**" since the time I created the discussion.

The issue I reported about the limited offering of issues remains just the same though.

I am referring to this menu:

![image](https://user-images.githubusercontent.com/8572152/160782783-cd73bc61-b1f5-46d3-b940-944620e2ccd6.png)

**Comment by machour** [comment]
I'm affected by this as well

**Comment by nul800sebastiaan** [comment]
Sometimes, I can retrospectively add `Fixed in #1234` but often that doesn't add a link either.
